.github/workflows/build.yml

```diff
@@ -28,3 +28,32 @@ jobs:
       - run: pnpm compile
       - run: pnpm lint
       - run: pnpm test
+
+  release:
+    runs-on: ubuntu-latest
+    needs: build
+    if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+    environment: Release
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v3
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: pnpm
+
+      - run: pnpm install --frozen-lockfile
+      
+      - name: Publish to Open VSX Registry
+        uses: HaaLeo/publish-vscode-extension@v1
+        with:
+          pat: ${{ secrets.OPEN_VSX_TOKEN }}
```

Comment:
Probably this first one needs ``dependencies: false` to avoid vsce/pnpm issue when building.
https://github.com/HaaLeo/publish-vscode-extension/tree/master?tab=readme-ov-file#input-parameters

Then for 2nd one, `.vsix` is already built during the 1st one, so we can use `extensionFile: ${{ steps.publishToOpenVSX.outputs.vsixPath }}` to reuse it.
https://github.com/HaaLeo/publish-vscode-extension?tab=readme-ov-file#example

Reply:
Fixed! ❤️